core/lib/spree/testing_support/kernel.rb

```diff
@@ -0,0 +1,18 @@
+module Spree
+  module TestingSupport
+    module Kernel
+
+      private
+
+      def silence_stream(stream)
+        old_stream = stream.dup
```

Comment:
@mauazua Good point. Thats why I added this as a private method. In fact, rails also added this method privately in few test files ([here](https://github.com/rails/rails/commit/481e49c64f790e46f4aff3ed539ed227d2eb46cb)). I have also added this method in install generator as its harmless there.